spec/models/family_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe Family do
+  describe "validations" do
+    context "is valid only if last_name and capability are present" do
+
```

Comment:
Extra newline.
Also line 5 is not needed.

Have you run Rubocop on this PR?

Reply:
Actually no, I was going to do it at the end but I suppose I should do more often (I still need to get used to it!).